Livecheckables/gtksourceviewmm3.rb

```diff
@@ -0,0 +1,4 @@
+class Gtksourceviewmm3
+  livecheck :url   => "https://download.gnome.org/sources/gtksourceviewmm/",
+            :regex => /gtksourceviewmm-(3\.([0-8]\d*?)?[02468](?:\.\d+)+)\.t/
```

Comment:
```suggestion
            :regex => /gtksourceviewmm-(3\.([0-8]\d*?)?[02468](?:\.\d+)*?)\.t/
```

This brings the regex in line with the `download.gnome.org` strategy's regex.